### DISCUSSION TITLE
Update about dialog

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#92] [General]
![](https://github.com/AdwCustomizerTeam/Design/raw/main/Screenshots/Screenshot%20from%202022-08-10%2014-18-32.png)

**Comment by daudix** [comment]
Looks good, is there will be version badge?

**Reply by 0xMRTT** [reply]
https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.AboutWindow.html

**Reply by ArtyIF** [reply]
My plans were to add one once the app becomes stable and properly versioned. I already pass the version parameter to the app, so really you have to add it in the about dialog and change it in meson.build
